src/components/Jam.jsx

```diff
@@ -172,29 +185,34 @@ export default function Jam() {
 
     setAlert(null)
     setIsLoading(true)
+    setIsWaitingSchedulerStop(true)
 
     const abortCtrl = new AbortController()
     return Api.getTakerStop({ signal: abortCtrl.signal, walletName: wallet.name, token: wallet.token })
       .then((res) => (res.ok ? true : Api.Helper.throwError(res, t('scheduler.error_stopping_schedule_failed'))))
-      .then((_) => setCollaborativeOperationRunning(false))
       .then((_) =>
         Promise.all([
           reloadServiceInfo({ signal: abortCtrl.signal }),
           reloadCurrentWalletInfo({ signal: abortCtrl.signal }),
         ])
       )
       .catch((err) => {
+        if (abortCtrl.signal.aborted) return
         setAlert({ variant: 'danger', message: err.message })
+        setIsWaitingSchedulerStop(false)
+      })
+      .finally(() => {
+        if (abortCtrl.signal.aborted) return
+        setIsLoading(false)
       })
-      .finally(() => setIsLoading(false))
   }
 
   return (
     <>
       <PageTitle title={t('scheduler.title')} subtitle={t('scheduler.subtitle')} />
       {alert && <rb.Alert variant={alert.variant}>{alert.message}</rb.Alert>}
 
-      {isLoading ? (
+      {isLoading || isWaitingSchedulerStart || isWaitingSchedulerStop ? (
         <rb.Placeholder as="div" animation="wave">
```

Comment:
Unrelated to this PR, but I thought I'd mention it anyway: I'm not sure how `animation="wave"` should look like, but this is how the loading state looks for me when starting/stopping the scheduler:

<img width="540" alt="Screenshot 2022-10-03 at 18 27 31" src="https://user-images.githubusercontent.com/109058/193630085-2656e10b-e492-4efa-8f25-b83811983215.png">

Independent of the changes in this PR, and independent of browser (Firefox and Chromium).